### ISSUE TITLE
[Mono] Tracking runtime test failures on Android arm32

### ISSUE THREAD

**Issue by fanyang-mono** [dotnet/runtime#53674]
These are part of the issues regarding to runtime test failures on Android arm32:

- [ ] #53638 [Mono][Android][arm32]JIT/Performance and JIT/CodeGenBringUpTests tests crashed when running on Android arm32
- [ ] #53633 [Mono][Android][arm32]Intrinsic test crashed when running on Android arm32

**Comment by steveisok** [comment]
@fanyang-mono Do you still plan on fixing the arm32 issue?

**Comment by fanyang-mono** [comment]
Yes, I will see if I have bandwidth to get to is for .NET7. Otherwise, I will move it to .NET8.

**Comment by fanyang-mono** [comment]
Moving to 8.0.0
